packages/layers/base/app/composables/useConnectLaunchDarkly.ts

```diff
@@ -0,0 +1,177 @@
+import { initialize } from 'launchdarkly-js-client-sdk'
+import type { LDClient, LDFlagSet, LDOptions, LDContext } from 'launchdarkly-js-client-sdk'
+
+// state
+// Defined outside of composable, so they are created only once and shared
+const ldClient = shallowRef<LDClient | null>(null)
+const ldFlagSet = shallowRef<LDFlagSet>({})
+const ldInitialized = ref(false)
+const isInitializing = ref(false)
```

Comment:
instead of ldInitialized and isInitializing do you think we could we set `ldClient = new LDClient()` at the beginning of 'init' and check if its null or not?